### ISSUE TITLE
Serve doc images from https://docs.mosaicml.com/en/stable

### ISSUE THREAD

**Issue by ravi-mosaicml** [mosaicml/composer#759]
Once v0.4.1 is merged into main, serve the doc assets from https://docs.mosaicml.com/en/stable rather than from the google storage bucket. It'd be better to keep everything in one place rather than split between the google storage bucket and the repository.

_Originally posted by @ravi-mosaicml in https://github.com/mosaicml/composer/pull/753#discussion_r827370611_

**Comment by ravi-mosaicml** [comment]
Closing as it is low-priority, and we'll run into this same issue whenever we want to update the readme.
